dev-util/mh/mh-1.0.1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="Generate help for Makefile"
+HOMEPAGE="https://github.com/oz123/mh"
+SRC_URI="https://github.com/oz123/mh/archive/refs/tags/v${PV}.tar.gz -> ${P}.tar.gz"
+LICENSE="BSD"
+SLOT="0"
+IUSE="test"
+KEYWORDS="~amd64"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="dev-libs/libpcre2"
+
+BDEPEND="test? ( dev-util/cmocka )"
+
+PATCHES="${FILESDIR}/Makefile.patch"
+
+src_compile() {
+	emake mh VERSION="${PV}"
+}
+
+src_install() {
```

Comment:
We can’t use your own install target?

is this because we need to fix the Makefile?

Reply:
Yes, I think so. I can fix the makefile upstream and try again.
@SoapGentoo @thesamesam thanks for the review.

Reply:
@thesamesam using my own target, the binaries get installed to `/usr/local` ...

Reply:
This is now fixed. I had to use PREFIX and DESTDIR (with some changes to the Makefile too).